src/promptflow/tests/sdk_cli_azure_test/conftest.py

```diff
@@ -36,7 +37,22 @@
 RESOURCE_ID_FORMAT = "/subscriptions/{}/resourceGroups/{}/providers/{}/workspaces/{}"
 
 
-@pytest.fixture
+def determine_scope() -> str:
```

Comment:
maybe rename this function to be more straight foward. when looking at determin_scope below, new devs might wonder what this means.

package_scope_in_live

Reply:
updated